components/index_embeddings/fondant_component.yaml

```diff
@@ -0,0 +1,43 @@
+name: Index embeddings
+description: Component that takes embeddings of text snippets and indexes them into a weaviate vector database.
+image: index_embeddings:latest
+
+consumes:
+  text:
+    fields:
+      data:
+        type: string
+      embedding:
+        type: array
+        items:
+          type: float32
+
+args:
+  weaviate_url:
+    description: The URL of the weaviate instance.
+    type: str
+    default: http://localhost:8080
+  model:
+    description: |
+      A name for the model used to generate the embeddings.
+      It is used to name the weaviate class and thus has to follow the naming conventions of weaviate.
+    type: str
+    default: MiniLM
+  dataset:
+    description: |
+      The name of the data source from which the text snippets that were used to generate the embeddings stem.
+      It is used to name the weaviate class and thus has to follow the naming conventions of weaviate.
+    type: str
+  vectorizer:
+    description: |
+      The configuration of the weaviate vectorizer (for vectorization with text2vec-* modules) in the class object 
+      corresponding to the model used to generate the embeddings. 
+      You can find the available modules in the weaviate documentation: https://weaviate.io/developers/weaviate/modules/retriever-vectorizer-modules. 
+      This can be used for smoother interaction with the weaviate instance after indexing e.g. using langchain similarity search. 
+      The module used also has to be enabled in your weaviate instance deployment. 
+      Can also be {"vectorizer": "none"} if you do not want to make use of this feature.
+    type: dict
+  overwrite:
+    description: Whether to overwrite/ re-create the existing weaviate class and its embeddings.
+    type: bool
+    default: 0
```

Comment:
I think it's better to set the default as a Boolean

Reply:
can I just write: False? if I recall correctly this wasnt possible when i tried it as it reads it can only read it as a str or int

Reply:
that is why I also cound not set a default for the vectorizer arg

Reply:
I think if you're on the latest branch it should work, we recently fixed something similar related to arguments. Could you try it and let me know if you run into other issues?